src/live.asp.net/Services/FeedService.cs

```diff
@@ -78,86 +78,73 @@ public AtomFeed()
         [XmlNamespaceDeclarations]
         public XmlSerializerNamespaces Namespaces { get; set; }
 
-        [XmlElement("id")]
-        public string Id { get; set; }
+        public string id { get; set; }
```

Comment:
Because making them lowercase allows me to remove the annotations where not needed. I can simply revert the commit if you want.